main.cpp

```diff
@@ -107,11 +108,59 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+    
+    void run(int, bool);
+    void stepForward();
+    float stepSize();
+
+    struct Foot{
+        int footSize;
+        int stepDistance;
+        int timeTaken;
+        void moveForward(int numSteps)
+        {
+            numSteps += 1;
+        }
+                
+        int stepSize()
+        {
+            return footSize + stepDistance; 
+        }
+        void stepForward(bool move = false)
+        {
+            if (move == true) moveForward(1);
+            return;
+        }
```

Comment:
no need for the `move == true`.  `move` is a boolean.
No need for the `return` statement, since the function's return type is `void`.
```
if( move )
    moveForward(1);
```    

- [ ] revise